.gitignore

```diff
@@ -158,3 +158,5 @@ cython_debug/
 #  and can be added to the global gitignore or merged into this file.  For a more nuclear
 #  option (not recommended) you can uncomment the following to ignore the entire idea folder.
 #.idea/
+test/
+pathogenEmbed.yml
```

Comment:
Why do we need to ignore these specific paths?

Reply:
I have a "test" repo that is filled with fasta files and stuff I used for testing the commands - didn't think it would make sense to push here! And the yml file is just not working yet so I thought I'd add it when it works as expected.